src/app/shared/nav-bar/nav-bar.styles.css

```
@@ -19,7 +19,7 @@
 }
 #nav-links li a {
   font-family: 'Dosis', sans-serif;
-  color: white;
+  color: rgb(252, 248, 248);
```

Why did you change the color here?
```suggestion
  color: white;
```